code/game/objects/effects/portals.dm

```diff
@@ -45,29 +47,24 @@
 	return ..()
 
 /obj/effect/portal/attackby(obj/item/W, mob/user, params)
-	if(user && Adjacent(user))
-		user.forceMove(get_turf(src))
+	if(user && (Adjacent(user) || get_turf(user) == get_turf(src)))
```

Comment:
```suggestion
	if(user && Adjacent(user))
```
The second check will never succeed, because the first one already succeeds on that condition.

Reply:
oops, yeah that makes sense. thanks